packages/auth/src/app.ts

```diff
@@ -219,13 +221,6 @@ export class App {
     // Token management
     this.publicRouter.post('/auth/introspect', accessTokenRoutes.introspect)
 
-    this.publicRouter.post('/auth/token/:id', (ctx: AppContext): void => {
-      // TODO: tokenService.rotate
-      ctx.status = 200
-    })
-
-    this.publicRouter.del('/auth/token/:id', accessTokenRoutes.revoke)
```

Comment:
I guess because we have this section further up that also includes checking against the openapi spec (L168 - L1206). In the end, all of those individual routes should be moved into the block that also checks the openapi spec.

Reply:
It is redundant to [this block of code now](https://github.com/interledger/rafiki/blob/16deb5bc3605e83594ac7673c8052a7b2a453a93/packages/auth/src/app.ts#L193-L242). I originally stubbed it out when scaffolding the auth server but it didn't get removed once OpenApi was added.